pkgs/git.yaml

```diff
@@ -19,6 +20,13 @@ build_stages:
   bash: |
     export PERL_PATH="/usr/bin/env perl"
 
+- when: platform == 'Darwin' and disable_crypto
+  name: fix_crypto_compile_failure
+  before: configure
+  handler: bash
+  bash: |
+    export NO_APPLE_COMMON_CRYPTO=1
```

Comment:
This disables https support, so it is not a solution. It makes `git` build with our gcc at least, so that's why I did it, but we need to find a real solution.